connectors/sink-tencentcloud-scf/internal/server.go

```diff
@@ -33,19 +32,26 @@ import (
 )
 
 const (
-	name = "Tencent Cloud COS Source"
+	name = "Tencent SCF Sink"
 )
 
 var (
 	functionNamePrefix = "vanus-cos-source-function"
 )
 
+var _ config.SinkConfigAccessor = &Config{}
+
 type Config struct {
+	config.SinkConfig
 	F      Function `json:"function" yaml:"function"`
 	Debug  bool     `json:"debug" yaml:"debug"`
 	Secret *Secret  `json:"-" yaml:"-"`
 }
 
+func (c Config) GetSecret() config.SecretAccessor {
+	return c.Secret
```

Comment:
secret is pointer, it zero value is nil ,can not set value

Reply:
DONE